CONTRIBUTING.md

```diff
@@ -2,20 +2,68 @@
 
 We welcome all contributions, suggestions, and feedback, so please do not hesitate to reach out!
 
-The easiest way to reach us is on the [Kubernetes slack #kyverno channel](https://app.slack.com/client/T09NY5SBT/CLGR9BJU9). 
 
 ## Ways you can contribute
-- Report potential bugs
-- Request a feature
-- Request a sample policy
-- Join our community meetings
-- Submit a PR for [open issues](https://github.com/kyverno/kyverno/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22)
-- Fix or improve documentation
+### Report issues
```

Comment:
Can we add a content table right after "## Ways you can contribute", just to provide an overview of the page? We can also add the hyperlink to each section, for example https://github.com/kyverno/kyverno/wiki/Running-in-development-mode.